### PR TITLE
Remove apollo-execution

### PR DESCRIPTION
**Pull request by martinbonnin**
`apollo-execution` has moved to https://github.com/apollographql/apollo-kotlin-execution/

### CODE REVIEW

File: libraries/apollo-annotations/api/apollo-annotations.klib.api [hunk 1]
```diff
@@ -46,18 +46,3 @@ open annotation class com.apollographql.apollo3.annotations/ApolloInternal : kot
...
-open annotation class com.apollographql.apollo3.annotations/GraphQLName : kotlin/Annotation { // com.apollographql.apollo3.annotations/GraphQLName|null[0]
-    constructor <init>(kotlin/String) // com.apollographql.apollo3.annotations/GraphQLName.<init>|<init>(kotlin.String){}[0]
-    final val name // com.apollographql.apollo3.annotations/GraphQLName.name|{}name[0]
-        final fun <get-name>(): kotlin/String // com.apollographql.apollo3.annotations/GraphQLName.name.<get-name>|<get-name>(){}[0]
-}
-open annotation class com.apollographql.apollo3.annotations/GraphQLObject : kotlin/Annotation { // com.apollographql.apollo3.annotations/GraphQLObject|null[0]
-    constructor <init>(kotlin/String = ...) // com.apollographql.apollo3.annotations/GraphQLObject.<init>|<init>(kotlin.String){}[0]
-    final val name // com.apollographql.apollo3.annotations/GraphQLObject.name|{}name[0]
-        final fun <get-name>(): kotlin/String // com.apollographql.apollo3.annotations/GraphQLObject.name.<get-name>|<get-name>(){}[0]
-}
```
Comment (martinbonnin) [inline libraries/apollo-annotations/api/apollo-annotations.klib.api:63]: Those annotations have moved as well

File: libraries/apollo-debug-server/graphql/schema.graphqls [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  records: [Record!]!
+}
+
+type Record {
+  key: String!
+
+  fields: Fields!
+
+  sizeInBytes: Int!
+}
```
Comment (martinbonnin) [inline libraries/apollo-debug-server/graphql/schema.graphqls:45]: New `apollo-execution` can generate a SDL file from the code itself (it's now truly code-first). This file is generated from the Kotlin code and checked in source control to monitor for API changes (just like BCV but for the network boundary)

Comment (BoD) [reply libraries/apollo-debug-server/graphql/schema.graphqls:45]: Super neat! 👍

Comment (martinbonnin) [reply libraries/apollo-debug-server/graphql/schema.graphqls:45]: Now that I think about it, there should be a "schemaDump" and "schemaCheck" task, just like BCV too, I'll think of something

**Review by BoD — APPROVED** [review]
🚀

**Pull request merged**